capa/main.py

```diff
@@ -965,11 +997,14 @@ def main(argv: Optional[list[str]] = None):
         ensure_input_exists_from_cli(args)
         input_format = get_input_format_from_cli(args)
         rules = get_rules_from_cli(args)
-        found_file_limitation = False
+        # logger.info(rules.rules)
+        found_limitation = False
+        file_extractors = get_file_extractors_from_cli(args, input_format)
         if input_format in STATIC_FORMATS:
             # only static extractors have file limitations
-            file_extractors = get_file_extractors_from_cli(args, input_format)
-            found_file_limitation = find_file_limitations_from_cli(args, rules, file_extractors)
+            found_limitation = find_file_limitations_from_cli(args, rules, file_extractors)
```

Comment:
lets rename this `find_static_limitations_from_cli` for symmetry with the dynamic case.